tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -24,6 +24,19 @@
 #include "tls/s2n_connection.h"
 #include "tls/s2n_security_policies.h"
 
+static int s2n_conn_set_chosen_psk(struct s2n_connection *conn) {
+    EXPECT_NOT_NULL(conn);
+
+    uint8_t psk_identity[] = "psk identity";
+    EXPECT_OK(s2n_psk_parameters_init(&conn->psk_params));
+    EXPECT_OK(s2n_array_pushback(&conn->psk_params.psk_list, (void**) &conn->psk_params.chosen_psk));
+    EXPECT_SUCCESS(s2n_psk_init(conn->psk_params.chosen_psk, S2N_PSK_TYPE_EXTERNAL));
+    EXPECT_NOT_NULL(conn->psk_params.chosen_psk);
+    EXPECT_SUCCESS(s2n_psk_new_identity(conn->psk_params.chosen_psk, psk_identity, sizeof(psk_identity)));
```

Comment:
Nit: I'd suggest GUARDs/ENSUREs instead of EXPECTs. With EXPECTs, it's harder to tell which test failed because the program bails on that exact line, inside this shared method.